controllers/release/release_adapter.go

```diff
@@ -197,45 +197,46 @@ func (a *Adapter) finalizeRelease() error {
 	return nil
 }
 
-// getActiveTargetReleaseLink returns the ReleaseLink targeted by ReleaseLink in the Release being processed.
-// Only ReleaseLinks with the auto-release label set to true (or missing the label, which is treated the same as
-// having the label and it being set to true) will be searched for. If a matching ReleaseLink is not found or
-// the List operation fails, an error will be returned.
-func (a *Adapter) getActiveTargetReleaseLink() (*v1alpha1.ReleaseLink, error) {
-	releaseLink, err := a.getReleaseLink()
+// getActiveReleaseAdmissionPlan returns the ReleaseAdmissionPlan targeted by the ReleasePlan in the Release being
+// processed. Only ReleaseAdmissionPlans with the 'auto-release' label set to true (or missing the label, which is
+// treated the same as having the label and it being set to true) will be searched for.
```

Comment:
It is weird having this line so short. Surely some of the next line can fit on it

Reply:
Fixed.